packages/enzyme/src/Utils.js

```diff
@@ -42,7 +42,7 @@ export function makeOptions(options) {
 }
 
 export function isCustomComponentElement(inst, adapter) {
-  return !!inst && adapter.isValidElement(inst) && typeof inst.type === 'function';
```

Comment:
Please add tests that ensure `isCustomComponentElement` returns `false` for HTML elements.

Reply:
Hi @ljharb,

I thing the changes are ready but fails the Travis job that build react 16.5 with node 8.

My develop machine use node 8.11.3 with Ubuntu 18.04 and I pass test:all and the lint tests.

Is it possible that it' s a specific problem with the Travis environment or is the problem in my code?

Thanks,

Reply:
Since it errored, instead of failed, it's an issue with travis.